lesson02/script.js

```diff
@@ -0,0 +1,12 @@
+const li = document.createElement('li');
+
+const a = document.createElement('a');
+a.href = '1.html';
+a.textContent = 'これです';
+
+const img = document.createElement('img');
+img.src = 'bookmark.png';
+img.alt = 'ブックマーク';
+
+const ul = document.getElementById('list');
```

Comment:
nits
It is recommended that IDs and classes that are the starting point for JavaScript be prefixed with .js- to clearly indicate that JavaScript is involved.
https://zenn.dev/haniwaman/articles/bf392f397c8db7341881

Reply:
Thank you for your advice!

Thanks to you, I understood that you need to prepend js- to the id and class related to js.
I will add js- to id.